### PR TITLE
✨ Build CourseDetail page

### PR DESCRIPTION
**Pull request by daebert**
- HTML and CSS
- functions to do

### CODE REVIEW

File: src/app/pages/CourseDetail/CourseDetail.module.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  grid-template-rows: auto 1fr;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  width: 90%;
```
Comment (roabram) [inline src/app/pages/CourseDetail/CourseDetail.module.css:13]: Why don't you give the container a width with 90% if main and header are inside?

**Review by roabram — APPROVED** [review]
lgtm 💯

File: src/app/pages/CourseDetail/CourseDetail.module.css (hunk 1 above)
Comment (daebert) [reply src/app/pages/CourseDetail/CourseDetail.module.css:13]: Thank you. There is a button in the footer of the page (without CSS styling) that has a with of 100%.

**Comment by phlgr** [comment]
Please resolve the merge conflict before marking the PR as ready to review.

**Pull request merged**